docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx

```diff
@@ -34,13 +34,13 @@ import {
 
 <AppDirectoryAlert />
 
-To use the StorageManager component import it into your React application with the included styles. 
+To use the FileUploader component import it into your React application with the included styles. 
```

Comment:
We should add an informational alert here that StorageManager is now FileUploader

Reply:
Done. Should I still keep the "Storage Manager is deprecated" warning alert in the Deprecated props section?